exercises/series/.meta/src/reference/java/Series.java

```diff
@@ -7,30 +7,35 @@
 class Series {
 
     private final int digitsSize;
-    private List<Integer> digits;
+    private List<String> digits;
 
     Series(String string) {
-        this.digits = Arrays.stream(string.split(""))
-                        .map(Integer::parseInt)
-                        .collect(Collectors.toList());
-        this.digitsSize = this.digits.size();
+        this.digits = Arrays.stream(string.split("")).collect(Collectors.toList());
+        if(!string.isEmpty()) {
+            this.digitsSize = this.digits.size();
+        }else {
```

Comment:
Very minor thing: there should be a space before `else` here 🙂 You could also write this using a ternary operator if you want:

```
this.digitsSize = string.isEmpty() ? this.digitz.size() : 0;
```

🙂